pom.xml

```diff
@@ -51,6 +51,7 @@
     <module>packaging/hudi-timeline-server-bundle</module>
     <module>docker/hoodie/hadoop</module>
     <module>hudi-integ-test</module>
+    <module>hudi-test-coverage-aggregator</module>
```

Comment:
is there a way to do this without adding this module?

Reply:
for now we cannot do without adding a new module in jacoco. there is a [PR](https://github.com/jacoco/jacoco/pull/1007) out in jacoco OSS to avoid this. But it is not getting much traction. i am closely following it. will remove this module once that PR gets merged to a release. for now this is just a wrapper module that does not get deployed.

Reply:
I really dislike the additional module. can you reuse one of the bundle modules instead?  or `hudi-integ-test`?